### ISSUE TITLE
Bold font variant not being loaded when forced to load all variants

### ISSUE THREAD

**Issue by reefki** [themeum/kirki#2122]
### Issue description:
As the title said, I tell Kirki to load all font variants using the code provided below, but it seems that whatever font I choose the Bold or 700 variant isn't loaded, only the italic version (700italic).

### Version used:
3.0.38

### Using theme_mods or options?
`theme_mods`

### Code to reproduce the issue (config + field(s))
```
function kirki_load_all_font_variants() {
    if (class_exists('Kirki_Fonts_Google')) {
        Kirki_Fonts_Google::$force_load_all_variants = true;
        Kirki_Fonts_Google::$force_load_all_subsets = true;
    }
}

add_action('init', 'kirki_load_all_font_variants');
```
```
Kirki::add_field('my_config', array(
    'type' => 'typography',
    'settings' => 'typography',
    'label' => esc_html__('Typography', 'textdomain'),
    'section' => 'typography',
    'transport' => 'auto',
    'default' => [
		'font-family' => 'Roboto',
		'variant' => false,
    ],
    'output' => array(
	array(
            'element' => 'body'
        )
    ),
));
```

**Comment by hogash** [comment]
Just encountered the same problem and found out it's caused by a strict `in_array` int vs string at https://github.com/aristath/kirki/blob/master/modules/webfonts/class-kirki-fonts-google.php#L216 , however just noticed Ari added a strval tweak in the develop branch https://github.com/aristath/kirki/blob/develop/modules/webfonts/class-kirki-fonts-google.php#L216 .

Thanks Ari!! 🥇

**Comment by aristath** [comment]
ah yes that has already been fixed and will be included in v3.0.39 in a few days (by Friday this will be out) so I'll go ahead and close this one.
